### ISSUE TITLE
About testing the face alignment without ground truth *.pts files

### ISSUE THREAD

**Issue by DoraXingyu** [jwyang/face-alignment#24]
Hi, your code has been very helpful to me! Thank you for your contribution!
I have got your code running with afw dataset and the result is good.
However, I meet some problems when I try to do face alignment with my own images which do not have the ground truth .pts files. 
I read through your code, and try to replace bbox_facedet with viola jones detector result. But the shape_gt variable seems confusing to me.
Could you tell me how can I modify your code to realise face alignment with my own images?

**Comment by jwyang** [comment]
Hi, thanks for using the codes.
The simplest way is replacing the shape_gt with arbitrary shape for your data that do not have groundtruth shape. You can modify the codes in loadsmaples.m. Specifically, loadshape(path). Asseert whether there is a pts file, if not, then replace with another fake shape, say the mean shape.

**Comment by DoraXingyu** [comment]
Yes, that solution helps. But when I set the shape_gt to mean_shape, the alignment seems to lose the ability to deform, but use only the mean_shape to try to fit the face. Do you have any ideas why this happens? Or, could you explain a bit about what the "sr- for loop" in "test_model.m" is doing?
<img width="554" alt="screen shot 2015-10-10 at 7 49 36 pm" src="https://cloud.githubusercontent.com/assets/11518107/10410991/18f66160-6f88-11e5-9e68-14a449ab72da.png">
